File: orderentry/src/main/java/com/secwager/orderentry/di/OrderEntryComponent.java
package com.secwager.orderentry.di;

import com.secwager.orderentry.OrderEntryServiceImpl;
import dagger.Component;
import javax.inject.Singleton;

@Singleton
@Component(modules = {OrderEntryModule.class})
public interface OrderEntryComponent {

  OrderEntryServiceImpl buildOrderEntryService();
}

File: orderentry-grpc/src/main/kotlin/di/OrderEntryComponent.kt
package com.secwager.orderentry.di

import com.secwager.orderentry.OrderEntryServiceImpl
import dagger.Component
import javax.inject.Singleton

@Singleton
@Component(modules = [OrderEntryModule::class])
interface OrderEntryComponent {
    fun buildOrderEntryService(): OrderEntryServiceImpl?
}